dist/hunt.js

```diff
@@ -111,6 +112,9 @@
   // fallback leave function to avoid sanity check
   Hunted.prototype.leave = noop;
 
+  // throttle interval
+  Hunted.prototype.throttleInterval = DEFAULT_THROTTLE_INTERVAL;
```

Comment:
I think this line is not necessary since we are actually not using this value when initializing the event listeners.

Reply:
@jeremenichelli You're right, I've pushed an update to my fork, it should come through here soon 👍